src/modules/polls.ts

```diff
@@ -134,25 +133,22 @@ export const scheduleAttendancePolls = async (
 
 			if (!channel) {
 				console.error(
-					`Couldn't fetch channel ${poll.channelId} for attendance poll ${poll.id}`
+					`Couldn't fetch channel ${poll.channelId} for poll ${poll.id}`
 				);
 				return;
 			}
 
-			cron.schedule(poll.cron, async () => {
+			cron.schedule(poll.cron as string, async () => {
 				try {
 					// make sure it wasn't deleted / edited in the meantime
-					const p = await prisma.attendancePoll.findFirst({
+					const p = await prisma.poll.findFirst({
 						where: { id: poll.id },
 					});
 					if (p !== null) {
-						await sendAttendanceEmbed(p, channel as TextChannel);
+						await sendPollEmbed(p, channel as TextChannel);
 					}
 				} catch (e) {
-					console.error(
-						"Could not verify (& send) attendance poll:",
-						e.message
-					);
+					console.error("Could not verify (& send) poll:", e.message);
```

Comment:
This is probably just me, but TS doesn't like this (`Object is of type unknown`), I guess since you can technically `throw` any value, not necessarily an `Error`
```suggestion
					console.error("Could not verify (& send) poll:", (e as Error).message);
```